### PR TITLE
Fix for all udid alternatives and linter check

### PR DESCRIPTION
**Pull request by commjoen**
Thank you for submitting a Pull Request to the Mobile Security Testing Guide. Please make sure that:

- [x] Your contribution is written in the 2nd person (e.g. you)
- [x] Your contribution is written in an active present form for as much as possible.
- [x] You have made sure that the reference section is up to date (e.g. please add sources you have used, make sure that the references to MITRE/MASVS/etc. are up to date)
- [x] Your contribution has proper formatted markdown and/or code
- [x] Any references to website have been formatted as [TEXT](URL “NAME”)
- [x] You verified/tested the effectiveness of your contribution (e.g.: is the code really an effective remediation? Please verify it works!)

If your PR is related to an issue. Please end your PR test with the following line:
This PR covers issue #< insert number here >.

### CODE REVIEW

**Comment by OMTGreleaser** [comment]
Results for commit 2b00d0dc6d5d0feb7398bef26ab344c85df722c0: Broken link result: [!] Error(s) found by the Links validator: 6 pages have dead links! Verbose output in /link-check-result.out .
 Please run the Apply_Link_Check.sh in the tools folder to see wherhe the Link issues are. 
 Markdown result errorlength:        0 ../linter-result.out .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

File: Document/0x06b-Basic-Security-Testing.md [hunk 1]
```diff
@@ -31,14 +31,36 @@ $ xcode-select --install
...
 The UDID is a 40-digit unique sequence of letters and numbers to identify an iOS device. You can find the [UDID of your iOS device via iTunes](http://www.iclarified.com/52179/how-to-find-your-iphones-udid "How to Find Your iPhone's UDID"), by selecting your device and clicking on "Serial Number" in the summary tab. When clicking on this you will iterate through different meta-data of the iOS device including its UDID.
 
-It is also possible to get the UDID via the command line, from a device attached via USB using `ioreg`:
+It is also possible to get the UDID via various commindline tools while the device is attached via USB:
 
-```shell
-$ ioreg -p IOUSB -l | grep "USB Serial"
-  |         "USB Serial Number" = "9e8ada44246cee813e2f8c1407520bf2f84849ec"
-```
+- By using `ioreg`:
```
Comment (sushi2k) [inline Document/0x06b-Basic-Security-Testing.md]: Is ioreg only macOS? If so, please mention it, like:

By using `ioreg` (macOS only):

File: Document/0x06b-Basic-Security-Testing.md [hunk 2]
```diff
@@ -31,14 +31,36 @@ $ xcode-select --install
...
+        |         "USB Serial Number" = "9e8ada44246cee813e2f8c1407520bf2f84849ec"
+    ```
+
+- From a tool called ideviceinstaller:
+
+    ```sh
+        $ brew install ideviceinstaller
+        $ idevice_id -l
+        316f01bd160932d2bf2f95f1f142bc29b1c62dbc
+    ```
```
Comment (sushi2k) [inline Document/0x06b-Basic-Security-Testing.md]: Can you please remove the whitespaced  from the commands above so it's left aligned?

    ```sh
    $ brew install ideviceinstaller
    $ idevice_id -l
    316f01bd160932d2bf2f95f1f142bc29b1c62dbc
    ```

File: Document/0x06b-Basic-Security-Testing.md [hunk 3]
```diff
@@ -31,14 +31,36 @@ $ xcode-select --install
...
-  |         "USB Serial Number" = "9e8ada44246cee813e2f8c1407520bf2f84849ec"
-```
+- By using `ioreg`:
+
+    ```sh
+        $ ioreg -p IOUSB -l | grep "USB Serial"
+        |         "USB Serial Number" = "9e8ada44246cee813e2f8c1407520bf2f84849ec"
+    ```
+
+- From a tool called ideviceinstaller:
```
Comment (sushi2k) [inline Document/0x06b-Basic-Security-Testing.md]: Change to:

By using `ideviceinstaller` (macOS and Linux):

File: Document/0x06b-Basic-Security-Testing.md [hunk 4]
```diff
@@ -31,14 +31,36 @@ $ xcode-select --install
...
+
+    ```sh
+        $ brew install ideviceinstaller
+        $ idevice_id -l
+        316f01bd160932d2bf2f95f1f142bc29b1c62dbc
+    ```
+
+- Using the systemprofiler:
```
Comment (sushi2k) [inline Document/0x06b-Basic-Security-Testing.md]: By using `system_profiler` (macOS only):

File: Document/0x06b-Basic-Security-Testing.md [hunk 5]
```diff
@@ -31,14 +31,36 @@ $ xcode-select --install
...
+    $ system_profiler SPUSBDataType | sed -n -e '/iPad/,/Serial/p;/iPhone/,/Serial/p;/iPod/,/Serial/p' | grep "Serial Number:"
+    2019-09-08 10:18:03.920 system_profiler[13251:1050356] SPUSBDevice: IOCreatePlugInInterfaceForService failed 0xe00002be
+                Serial Number: 64655621de6ef5e56a874d63f1e1bdd14f7103b1
+    ```
+
+- Using instruments:
 
-Alternatively you can also use the Xcode command `instruments -s devices`.
+    ```sh
+        $ nstruments -s devices
```
Comment (sushi2k) [inline Document/0x06b-Basic-Security-Testing.md]: nstruments -> instruments

**Comment by OMTGreleaser** [comment]
Results for commit 12667d641491540a0af92fd6c6701042c79deb11: Broken link result: [!] Error(s) found by the Links validator: 6 pages have dead links! Verbose output in /link-check-result.out .
 Please run the Apply_Link_Check.sh in the tools folder to see wherhe the Link issues are. 
 Markdown result errorlength:        0 ../linter-result.out .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

**Comment by OMTGreleaser** [comment]
Results for commit 5d29ba680b9054f8d5ce8b5f908a5d5046709b52: Broken link result: [!] Error(s) found by the Links validator: 5 pages have dead links! Verbose output in /link-check-result.out .
 Please run the Apply_Link_Check.sh in the tools folder to see wherhe the Link issues are. 
 Markdown result errorlength:        0 ../linter-result.out .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

File: Document/0x06b-Basic-Security-Testing.md [hunk 6]
```diff
@@ -31,14 +31,36 @@ $ xcode-select --install
...
 The UDID is a 40-digit unique sequence of letters and numbers to identify an iOS device. You can find the [UDID of your iOS device via iTunes](http://www.iclarified.com/52179/how-to-find-your-iphones-udid "How to Find Your iPhone's UDID"), by selecting your device and clicking on "Serial Number" in the summary tab. When clicking on this you will iterate through different meta-data of the iOS device including its UDID.
 
-It is also possible to get the UDID via the command line, from a device attached via USB using `ioreg`:
+It is also possible to get the UDID via various command line tools while the device is attached via USB:
 
-```shell
-$ ioreg -p IOUSB -l | grep "USB Serial"
-  |         "USB Serial Number" = "9e8ada44246cee813e2f8c1407520bf2f84849ec"
-```
+-  By using the [I/O Registry Explorer](https://developer.apple.com/library/archive/documentation/DeviceDrivers/Conceptual/IOKitFundamentals/TheRegistry/TheRegistry.html "I/O Registry Explorer") tool `ioreg` (macOS only):
```
Comment (cpholguera) [inline Document/0x06b-Basic-Security-Testing.md]:
```suggestion
- By using the [I/O Registry Explorer](https://developer.apple.com/library/archive/documentation/DeviceDrivers/Conceptual/IOKitFundamentals/TheRegistry/TheRegistry.html "I/O Registry Explorer") tool `ioreg` (macOS only):
```

**Comment by OMTGreleaser** [comment]
Results for commit 26594ebd494a04ac8614a190e39a7f7a93643642: Broken link result: [!] Error(s) found by the Links validator: 6 pages have dead links! Verbose output in /link-check-result.out .
 Please run the Apply_Link_Check.sh in the tools folder to see wherhe the Link issues are. 
 Markdown result errorlength:        0 ../linter-result.out .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

**Pull request merged**